pyke/kepextract.py

```diff
@@ -579,6 +594,10 @@ def kepextract_main():
                         type=str)
     parser.add_argument('--bkg', action='store_true',
                         help='Subtract background from data?')
+    parser.add_argument('--psfphot', action='store_true',
+                        help='Perform PSF Photometry with 2D Gaussian?')
```

Comment:
The option name `--psfphot` is misleading because we are not actually giving the user PSF photometry (= brightnesses), instead we are only measuring the centroids (= positions).

For this reason, I propose we name this option `--psfcentroid`, with help text `help="Measure the star's position by fitting a 2D Gaussian PSF to the pixels in the mask. This will populate values for PSF_CENTR1 (column position) and PSF_CENTR2 (row position) in the output file."`